
=== Test project_root: /tmp/pytest-of-runner/pytest-0/project23


Invoking dbt with ['run']
[0m12:21:30  Running with dbt=1.7.18
[0m12:21:30  Registered adapter: firebolt=1.6.4
[0m12:21:30  Unable to do partial parsing because saved manifest not found. Starting full parse.
[0m12:21:31  Found 1 model, 0 sources, 0 exposures, 0 metrics, 432 macros, 0 groups, 0 semantic models
[0m12:21:31  
[0m12:21:32  Concurrency: 1 threads (target='default')
[0m12:21:32  
[0m12:21:32  1 of 1 START sql view model model .............................................. [RUN]
[0m12:21:33  1 of 1 OK created sql view model model ......................................... [[32mSUCCESS[0m in 0.81s]
[0m12:21:33  
[0m12:21:33  Finished running 1 view model in 0 hours 0 minutes and 1.72 seconds (1.72s).
[0m12:21:33  
[0m12:21:33  [32mCompleted successfully[0m
[0m12:21:33  
[0m12:21:33  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['--warn-error', 'docs', 'generate']
[0m12:21:33  Running with dbt=1.7.18
[0m12:21:33  Registered adapter: firebolt=1.6.4
[0m12:21:33  Found 1 model, 0 sources, 0 exposures, 0 metrics, 432 macros, 0 groups, 0 semantic models
[0m12:21:33  
[0m12:21:33  Concurrency: 1 threads (target='default')
[0m12:21:33  
[0m12:21:33  Building catalog
[0m12:21:33  Encountered an error:
Compilation Error
  Encountered an error while generating catalog: Runtime Error
    Compilation Error in macro get_catalog_relations (macros/failure.sql)
      rejected: no catalogs for you
      
      > in macro get_catalog_relations (macros/failure.sql)
      > called by macro get_catalog_relations (macros/failure.sql)
